mitiq/shadows/shadows_utils.py

```diff
@@ -36,10 +97,8 @@ def local_clifford_shadow_norm(obs: mitiq.PauliString) -> float:
     """
     Calculate shadow norm of an operator with random unitary sampled from local
     Clifford group.
-
     Args:
-        obs: An observable in terms of a ``mitiq.PauliString`` object.
-
+        opt: a self-adjoint operator.
```

Comment:
Is this change intended? I added a commit (https://github.com/unitaryfund/mitiq/pull/1907/commits/a6c579348fe6c55b0291dce80a3012d0106c64a2) in https://github.com/unitaryfund/mitiq/pull/1907/ to fix this.